arbnode/batch_poster.go

```diff
@@ -156,6 +156,8 @@ type BatchPosterConfig struct {
 	L1BlockBoundBypass             time.Duration               `koanf:"l1-block-bound-bypass" reload:"hot"`
 	UseAccessLists                 bool                        `koanf:"use-access-lists" reload:"hot"`
 	GasEstimateBaseFeeMultipleBips arbmath.Bips                `koanf:"gas-estimate-base-fee-multiple-bips"`
+	ReorgResistanceMargin          time.Duration               `koanf:"reorg-resistance-margin" reload:"hot"`
+	IgnoreReorgResistanceMargin    bool                        `koanf:"ignore-reorg-resistance-margin" reload:"hot"`
```

Comment:
Optional(Nit): Prefer using boolean flags with a positive phrasing. Maybe "EnableReorgResistanceMargin" or "EnforceReorgResistanceMargin". When you have a negative like "Ignore" or "Disable" it means that you have to mentally negate the meaning of the boolean which slightly deteriorates the readability of the code.